tiledb/sm/storage_manager/storage_manager.h

```diff
@@ -1187,6 +1187,9 @@ class StorageManager {
 
   /** Sets default tag values on this StorageManager. */
   Status set_default_tags();
+
+  inline bool partial_time_overlap(
```

Comment:
Is this used?

Reply:
I don't know how this ended up here 😨 
removing!